ompi/debuggers/ompi_msgq_dll.c

```diff
@@ -1165,7 +1165,7 @@ static int fetch_request( mqs_process *proc, mpi_process_info *p_info,
             // data_name in res->extra_text[2] (vs. extra_text[1]),
             // where it is guaranteed to fit.
             data_name[4] = '\0';
-            snprintf( (char*)res->extra_text[1], 64, "Data: %d",
+            snprintf( (char*)res->extra_text[1], 64, "Data: %d instances of MPI datatype",
```

Comment:
**NOTE:** Per https://github.com/open-mpi/ompi/pull/9813#discussion_r777259766, I have reached out to Totalview and DDT to see if they actually parse the text in these strings at all.  If they don't, we'll keep this modification.  If they do, we'll have to revert it back to what it was before 8f2620d3af2.

Reply:
Totalview confirmed that they do not parse the messages (and confirmed that the documentation of the MQD API says that debuggers should not be parsing those messages).

I confirmed in https://www.mpi-forum.org/docs/msgq.5.pdf in section 5.5.5 p15, it says:

> `extra_text` is an array of five null-terminated strings that can be used by the MQD DLL to provide more information to the user. The debugger does not interpret this field and can display it to the user. This field can be used to give the name of the function causing this request, for example.

So I think we're good to go to merge this PR and not wait for an answer from DDT.